__tests__/app.test.js

```diff
@@ -170,3 +170,55 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: should post a new comment", () => {
```

Comment:
i would update this description to say also returns the posted comment as you are checking the structure of the returned comment in your test too